crates/typst-utils/src/format.rs

```diff
@@ -0,0 +1,237 @@
+//! Different formatting functions.
+
+use std::time::Duration;
+
+/// Returns value with `n` digits after floating point where `n` is `precision`.
+/// Standard rounding rules apply (if `n+1`th digit >= 5, round up).
+///
+/// If rounding the `value` will have no effect (e.g., it's infinite or
+/// NaN), returns `value` unchanged.
```

Comment:
Should reflow here to fill the 80 columns (it's possible this may be necessary elsewhere too)

```suggestion
/// If rounding the `value` will have no effect (e.g., it's infinite or NaN),
/// returns `value` unchanged.
```

Reply:
I'll check later.

Reply:
Oh, you mean _fill_, I thought of breaking down at first. Yeah, I don't know how that one got like this, probably after some rephrasing.

[Here](https://github.com/typst/typst/blob/a29af26746618b94b54272722d29cd217a884509/crates/typst-utils/src/format.rs#L81-L89) I tried to not finish the lines with preposition, breaking visually the linked/dependent words. I think that's the only place where the text is not fully filled.